src/layouts/Images.jsx

```diff
@@ -75,26 +104,23 @@ export default class Images extends Component {
   onImageSelect = async (event) => {
     const imgReader = new FileReader();
     imgReader.imgName = event.target.files[0].name;
```

Comment:
What is the point of setting `imgReader.imgName` and then reading from it in line 114? We can simply set `const imgName = event.target.files[0].name` and remove line 114

Reply:
yeah this is bad code my bad i did this because of some earlier implementation that cause funky scoping issues. Had changed the implementation but didn't know I could switch that out as well.